odl/util/testutils.py

```diff
@@ -317,7 +318,8 @@ def fail(self, string=None):
 
     def __exit__(self, type, value, traceback):
         if self.num_failed == 0:
-            print('{:<70}: Completed all test cases.'.format(self.test_name))
+            self.log('{:<70}: Completed all test cases.'
+                     ''.format(self.test_name))
         else:
             print(self.test_name)
```

Comment:
This `else` is the error case, I think it should print then regardless of verbosity.